### PR TITLE
feat: timer improved

### PR DESCRIPTION
**Pull request by AlefrankM**

### CODE REVIEW

File: src/pages/sessions/[interviewId].tsx [hunk 1]
```diff
@@ -101,15 +111,30 @@ export default function InterviewSessionPage() {
...
     } else {
       clearInterval(interval)
     }
+
+    socket.on('timer_started', (data) => {
+      setTime(data)
+    })
+
+    socket.on('timer_stopped', (data) => {
+      console.log(data)
```
Comment (juliswer) [inline src/pages/sessions/[interviewId].tsx:125]: remove clg

File: src/pages/sessions/[interviewId].tsx [hunk 2]
```diff
@@ -101,15 +111,30 @@ export default function InterviewSessionPage() {
...
+    })
+
+    socket.on('timer_cleared', (data) => {
+      console.log(data)
```
Comment (juliswer) [inline src/pages/sessions/[interviewId].tsx:129]: remove clg

**Review by juliswer — DISMISSED** [review]
Looks Good. Please remove clgs before merging

File: src/common/hooks/useSocket.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+import 'react-toastify/dist/ReactToastify.css'
+import io, { type Socket } from 'socket.io-client'
+
+const socket = io('ws://localhost:81', { transports: ['websocket'] })
```
Comment (juliswer) [inline src/common/hooks/useSocket.ts:4]: is it okay? It's linking with the localhost

**Review by juliswer — APPROVED** [review]
Looks good

File: src/common/hooks/useSocket.ts (hunk 3 above)
Comment (AlefrankM) [reply src/common/hooks/useSocket.ts:4]: Ohh shut, let me see

**Pull request merged**